src/main/java/com/epam/service/UserDetailsServiceImpl.java

```diff
@@ -22,7 +23,7 @@
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
         User user = userService.getUser("user");
         Set<GrantedAuthority> roles = new HashSet<>();
-        roles.add(new SimpleGrantedAuthority(user.getRole().getEnumrole()));
+        roles.add(new SimpleGrantedAuthority (UserRoleEnum.USER.getEnumrole()));
```

Comment:
Why u hardcode `userRole` here if it should depend on the user from DB?

Reply:
I answered on the same question in another pull request, but this changes additionaly provides me to resolve problem with doubled "getRole" calls.

Reply:
Doubled getRole calls...?
But in any case, I think it unnecessary movement, 'cause in our next pull request Lev fixed Security and there it's not hardcode